go/perf/codec-perf-rig/main.go

```diff
@@ -168,9 +168,8 @@ func buildListIncrementally(count uint64, createFn createValueFn) types.Collecti
 }
 
 func readList(c types.Collection) {
-	var outValue types.Value
 	c.(types.List).IterAll(func(v types.Value, idx uint64) {
-		outValue = v
+		_ = v
```

Comment:
Can you provide a comment in these three functions explaining why the bodies aren't just empty?